disbatch/createDisBTasks.py

```diff
@@ -0,0 +1,78 @@
+import glob, json, os.path, pandas, sys
+
+INPUT_DIR  = '/mnt/ceph/users/humanbase/data/meta/datasets/'
+TASKOUT_DIR = '/mnt/ceph/users/yliu/projects/MetaSRA-pipeline/data/'
```

Comment:
The output dir should probably be the current working directory of the repo clone, i.e. somewhere not specific to your ceph home dir. As part of this script, you can create the necessary directories.